app/src/main/java/app/insti/TagClass.java

```diff
@@ -0,0 +1,49 @@
+package app.insti;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+/**
+ * Created by Shivam Sharma on 13-08-2018.
+ */
+public class TagClass {
+
+    private String name;
+    private String color;
+
+    public TagClass(String name) {
+        this.name = name;
+        this.color = getRandomColor();
+    }
+
+    public String getRandomColor() {
+        ArrayList<String> colors = new ArrayList<>();
+        colors.add("#ED7D31");
+        colors.add("#00B0F0");
+        colors.add("#FF0000");
+        colors.add("#D0CECE");
+        colors.add("#00B050");
```

Comment:
All of these should be in resources

Reply:
ok. But they won't be changing in the future so it doesn't matter

Reply:
No, we can't rely on "being sure it won't change in the future"

Reply:
It will conflict with the design styles of the app

Reply:
Make a new resources file with the colors and get those here. How would this conflict?

Reply:
Sajal said not to change anything in the colors.xml file

Reply:
Yeah, don't. Make a new xml file.

Reply:
ok

Reply:
I don't recall saying that

Reply:
Then is it fine if i add these in color.xml?

Reply:
Then is it fine if i add these in color.xml?

Reply:
I asked you to not use colours that don't match the app theme, and I'm still saying that. I don't know where you're using these colours as I've not gone through your code yet, but if you just moved these colours to a new file and are still using them as theme colours in your module, then you'll have to change this again.
I repeat, changing colors.xml is not a problem, not following the app theme is.

Reply:
Ohk. Got it 👍